src/configs/getAndroidConfig.ts

```diff
@@ -134,6 +180,27 @@ export function getAndroidConfig(theme: Theme): Platform {
           isInternal: true,
         }),
       },
+      {
+        format: "compose/semantic-colors",
+        destination: `SemanticColors${className}.kt`,
+        className: className + "ColorTokens",
+        packageName: packageName,
+        filter: function (token: TransformedToken) {
+          return token.type == "color" && isNotCoreColor.matcher(token);
+        },
+        options: withDefaultOptions({
+          outputReferences: true,
+          import: [
+            "io.element.android.compound.annotations.CoreColorToken",
+            "io.element.android.compound.tokens.generated.internal." + tokenClassName
+          ],
+          isInternal: true,
```

Comment:
This is not used as far as I can tell.

Reply:
Did not want to touch this. Let's keep it for now and do a clean up later 👍